src/main/java/com/lootfood/api/controller/CityController.java

```diff
@@ -0,0 +1,64 @@
+package com.lootfood.api.controller;
+
+import com.lootfood.api.Dto.CityDto;
+import com.lootfood.entity.City;
+import com.lootfood.service.CityService;
+import com.mongodb.client.model.geojson.Position;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.geo.Point;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.stream.Collectors;
+
+import static com.lootfood.api.transformer.CityTransformer.transform;
+
+@RestController
+@RequestMapping("/api/city")
```

Comment:
Change url to `/api/lootfood/...` for all controllers `/api/lootfood/city` will be here